tests/scm/CMakeLists.txt

```diff
@@ -37,3 +37,22 @@ SET_PROPERTY(TEST SCMOpenogTestRunnerFail PROPERTY WILL_FAIL TRUE)
 # executing "make install"
 SET_PROPERTY(TEST SCMOpenogTestRunnerPass SCMOpenogTestRunnerFail
 	PROPERTY ENVIRONMENT "GUILE_LOAD_PATH=${PROJECT_SOURCE_DIR}/opencog/scm")
+
+# And some more tests
+IF (HAVE_CYTHON)
+	ADD_GUILE_TEST(SCMOpenogPython scm-opencog-python.scm)
+
+	# Debug prints ...
+	MESSAGE(STATUS "Project source dir: ${PROJECT_SOURCE_DIR}")
+	EXECUTE_PROCESS(COMMAND guile -c "(display %load-path)" OUTPUT_VARIABLE GLP)
+	MESSAGE(STATUS "Guile load path: ${GLP}")
+	SET(ENV{GUILE_LOAD_PATH} ${PROJECT_SOURCE_DIR}/opencog/scm:${PROJECT_SOURCE_DIR}/opencog/scm/opencog)
+	EXECUTE_PROCESS(COMMAND guile -c "(display %load-path)" OUTPUT_VARIABLE GLF)
+	MESSAGE(STATUS "Full guile load path: ${GLF}")
+	# EXECUTE_PROCESS(COMMAND ls -la ${PROJECT_SOURCE_DIR}/opencog/scm OUTPUT_VARIABLE SDL)
+	# MESSAGE(STATUS "Scheme directory contents: ${SDL}")
+	SET_PROPERTY(TEST SCMOpenogPython
+		PROPERTY ENVIRONMENT "GUILE_LOAD_PATH=${PROJECT_SOURCE_DIR}/opencog/scm:${PROJECT_SOURCE_DIR}/opencog/scm/opencog")
```

Comment:
PROJECT_SOURCE_DIR/opencog/scm doesn't have the write structure for modules. It should be PROJECT_BINARY_DIR/opencog/scm.

Reply:
trying now

Reply:
Thanks @AmeBel but that is still not working

Reply:
I rebased this pr on the branch of https://github.com/opencog/atomspace/pull/1975 and it is passing